impl/src/main/java/org/jboss/weld/event/ObserverMethodImpl.java

```diff
@@ -131,8 +131,8 @@ protected ObserverMethodImpl(final EnhancedAnnotatedMethod<T, ? super X> observe
             }
             injectionPoints.add(injectionPoint);
         }
-        this.injectionPoints = immutableGuavaSet(injectionPoints);
-        this.newInjectionPoints = immutableGuavaSet(newInjectionPoints);
+        this.injectionPoints = immutableSet(injectionPoints);
```

Comment:
`ImmutableSet.Builder` for `injectionPoints`. `newInjectionPoints` seem to be unsued - remove pls.